packages/ui/src/components/modals/CallbackResultViewerModal.css

```diff
@@ -1,11 +1,44 @@
+.cl-callback-result-modal__fields {
+    display: grid;
+    grid-gap: 1rem;
+}
+
 .cl-callback-result-modal__name {
```

Comment:
Empty style.   Any reason not to remove this?

Reply:
Hmm, I generally prefer this so the modal structure is setup for styling and next person doesn't have to establish selector name and possible DOM changes.  Especially since most other people don't follow BEM and do it wrong, but I also usually think its confusing to have unused code. Removed it.